frontend/leads-list/table/desktop-leads-table.component.tsx

```diff
@@ -0,0 +1,220 @@
+import React from "react";
+import { useCss } from "kremling";
+import dateformat from "dateformat";
+import { LeadsTableProps } from "./leads-table.component";
+import { formatPhone } from "../../util/formatters";
+import targetImg from "../../../icons/148705-essential-collection/svg/target.svg";
+
+export default function DesktopLeadsTable(props: LeadsTableProps) {
+  const scope = useCss(css);
+
+  return (
+    <div className="table-container" {...scope}>
+      <table className="leads-table">
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>Name</th>
+            <th>Phone</th>
+            <th>Event</th>
+            <th>Signup Date</th>
+            <th>Status</th>
+            <th>Interests</th>
+          </tr>
+        </thead>
+        <tbody
+          role="group"
+          aria-label="Select one or more lead for batch action"
+        >
+          {props.leads.length === 0 && !props.fetchingLeads ? (
+            <tr className="empty-state">
+              <td colSpan={7}>
+                <div>
+                  <img src={targetImg} alt="No leads" title="No leads" />
+                  <p>No clients match the search criteria</p>
+                </div>
+              </td>
+            </tr>
+          ) : (
+            props.leads.map(lead => {
+              console.log(lead);
+              return (
+                <tr key={lead.id}>
+                  <td>{lead.id}</td>
+                  <td>{lead.fullName}</td>
+                  <td>{formatPhone(lead.phone)}</td>
+                  <td className="capitalize">
+                    {lead.eventSources[lead.eventSources.length - 1].eventName}
+                  </td>
+                  <td>{dateformat(lead.dateOfSignUp, "m/d/yyyy")}</td>
+                  <td className="status-cell">
+                    {lead.leadStatus === "convertedToClient"
+                      ? `Converted to client - see client #${lead.clientId}`
```

Comment:
The nested ternaries!!!!! Ha I tweeted about this yesterday: https://twitter.com/Joelbdenning/status/1187149877978886144

Much better in my opinion would be to pull this out into its own function and then use if/else or switch/case:

In the jsx:
```tsx
{showLeadStatus(lead)}
```

Further down in the file:
```tsx
function showLeadStatus(lead) {
  if (lead.leadStatus === "convertedToClient") {
    return "..."
  } else if (lead.leadStatus === "inactive") {
  } else {
  }

  // etc etc etc
}
```